src/rust/inetstack/protocols/tcp/passive_open.rs

```diff
@@ -327,6 +330,7 @@ impl SharedPassiveSocket {
                 .clone()
                 .wait_for_ack(
                     recv_queue.clone(),
+                    ack_queue.clone(),
```

Comment:
Also do not need a clone here.

Reply:
Actually it does, because it is in a look and value would be moved.

Reply:
ah good point.